build.gradle

```diff
@@ -13,7 +13,10 @@ sourceCompatibility = 1.8
 
 dependencies {
     // This dependency is found on compile classpath of this component and consumers.
-    implementation 'org.checkerframework:checker:2.8.2git'
+//    implementation 'org.checkerframework:checker:2.8.2'
+    compile files('E:/gsoc-2019/The-Checker-Framework/checker-framework/checker/dist/checker-qual.jar')
```

Comment:
You should not commit local directories to the general configuration.
You can add a flag to run with a local CF Build, like you did for the template checker.

Reply:
@wmdietl, there is some issue in windows so I have to give an explicit path to the jar files.